components/table/src/Table.js

```diff
@@ -20,15 +20,15 @@ const TableWrapper = ({ bordered, responsive, withinPanel, ...props }) => {
 	return responsive ? (
 		<div
 			css={[
-				withinPanel && { border: 0, marginBottom: 0 },
+				extraStyles,
 				{
 					[xsOnly]: {
 						width: '100%',
-						marginBottom: withinPanel ? 0 : '1.125rem',
+						marginBottom: '1.125rem',
 						overflowY: 'hidden',
 						overflowX: 'auto',
 						// -ms-overflow-style: '-ms-autohiding-scrollbar',
```

Comment:
Need this?

Reply:
I think by keeping this in - the border will be removed *every time* the Table or PanelTable has the `responsive` prop. 

My understanding is that we only want to remove the border for the `PanelTable`, so I got rid of this ternary, and applied more specific styles in the `PanelTable` component (`!important).

Had to go more specific otherwise the media query wins the specificity battle, since we're not using a classname anymore.

I think it's a correct, intended use of `!important` so it makes it ok :)

Thoughts?

Reply:
referring to commented out `// -ms-overflow-style: '-ms-autohiding-scrollbar',`

Reply:
oh! no we don't